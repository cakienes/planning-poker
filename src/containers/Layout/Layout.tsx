import React from 'react';

const Layout: React.FC = ({ children }) => {
    return (
        <div className="container-fluid">
            <div className="row ">
                <div className="col">{children}</div>
            </div>
        </div>
    );
};

export default Layout;
