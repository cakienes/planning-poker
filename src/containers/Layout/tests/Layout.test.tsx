import { shallow } from 'enzyme';
import React from 'react';
import Layout from '../Layout';

describe('Layout.tsx', () => {
    it('renders without crashing', async () => {
        expect(shallow(<Layout />)).toMatchSnapshot();
    });
});
