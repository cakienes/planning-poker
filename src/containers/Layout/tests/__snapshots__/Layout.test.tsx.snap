// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout.tsx renders without crashing 1`] = `ShallowWrapper {}`;
